pkg/runtime/runtime.go

```diff
@@ -2229,12 +2225,9 @@ func (a *DaprRuntime) createAppChannel() error {
 			log.Infof("app max concurrency set to %v", a.runtimeConfig.MaxConcurrency)
 		}
 
-		// TODO: remove once feature is ratified
-		if httpCh, ok := ch.(*http_channel.Channel); ok {
```

Comment:
This warning should be shown only if we're using the HTTP server. No point in showing it when using gRPC.

Reply:
Ah yes, I can add that condition.